code/modules/projectiles/guns/energy/laser/antebellum.dm

```diff
@@ -14,7 +14,7 @@
 	suitable_cell = /obj/item/cell/medium/neotheology
 	projectile_type = /obj/item/projectile/plasma/light
 	fire_delay = 10
-	charge_cost = 100 //10 shots with basic 15 with level 2
+	charge_cost = 125 //10 shots with basic 15 with level 2
```

Comment:
please update the commit

Reply:
my bad! updating it! and the others too, I think there is more of those 🤔

Reply:
fixed